wkconnect/backends/tiff/models.py

```diff
@@ -97,52 +100,84 @@ def clear_cache(self) -> None:
     def read_properties(
         self, layer_name: str
     ) -> Tuple[
-        List[Vec3D], Tuple[int, int], List[Tuple[int, int]], np.dtype, List[List[int]]
+        List[Vec3D],
+        List[Tuple[int, int]],
+        List[Tuple[int, int]],
+        np.dtype,
+        List[List[int]],
     ]:
+        # Accessing the tiff header tags via their numbers:
+        tag_tile_width = 322
+        tag_tile_height = 323
+        tag_tile_byte_offsets = 324
+        tag_page_width = 256
+        tag_page_height = 257
```

Comment:
Maybe use capital-letters to indicate they are constants?
```suggestion
        TAG_TILE_WIDTH = 322
        TAG_TILE_HEIGHT = 323
        TAG_TILE_BYTE_OFFSETS = 324
        TAG_PAGE_WIDTH = 256
        TAG_PAGE_HEIGHT = 257
```

Or maybe directly use `TAGS()` from `tifffile`?
See https://github.com/cgohlke/tifffile/blob/master/tifffile/tifffile.py#L9413